utils/preview.ts

```diff
@@ -0,0 +1,37 @@
+import * as express from 'express';
+
+// Types
+import { Request, Response } from 'express';
+
+// Modules
+import * as fs from 'fs';
+import * as path from 'path';
+import * as glob from 'glob';
+
+// App
+const app = express();
+const port: number = 3000;
+
+const QUICKSTART_PATH: string = process.argv[2];
+
+/*
+  Validates that the directory path provided exists
+*/
+fs.access(`../quickstarts/${QUICKSTART_PATH}`, (error: any) => {
+  if (error) {
+    throw new Error(`Path: ${QUICKSTART_PATH} does not exist.`);
+  }
+});
+
+app.use(express.static(path.join(path.resolve(__dirname, '..'), `/quickstarts/${QUICKSTART_PATH}`)));
+
+app.get('/', (req: Request, res: Response) => {
+  let response = glob.sync(path.join(path.resolve(__dirname, '..'), `/quickstarts/${QUICKSTART_PATH}/**/*.*`))
+                     .map(filePath => filePath.slice(filePath.indexOf('/quickstarts')));
+                     
+  res.send(response);
+});
+
+app.listen(port, () => {
+  console.log("\x1b[32m", `Use this to get a list of your files! http://localhost:3000`);
```

Comment:
```suggestion
  console.log("\x1b[32m", `Use this to get a list of your files! http://newrelic.com/instant-observability/preview?local=true`);
```
I think it might be a good idea to have a link to I/O here so when the contributor starts their server, they have a link to click on that will send them to their preview, since it is going to be the browser that fetches from `localhost:3000`.

Reply:
Like we discussed on call Friday, updated the output to include the localhost port for the file output as well as the preview link to local I/O.  That will just need to be updated to the hosted I/O env when the feature is complete.